docs/resources/service_object.md

```diff
@@ -168,6 +168,16 @@ port {
 * `port` - (Required) This can be specified multiple times if the service can run on multiple ports. Structure is [documented below](#forwardproxy-port)
 * `tls_profile` - (Optional) Decryption profile id.
 
+## Forwarding
```

Comment:
@ganeshvaltix i see there is a forwarding service object already in the file. Can you please check that and updated that block.